plugins/email_service/app/views/email_service/email_verifications/new.html.haml

```diff
@@ -5,7 +5,9 @@
 = simple_form_for @verified_email, as: :verified_email, url: plugin('email_service').email_verifications_path, method: :post, remote: request.xhr?, html: {data: {modal: true}, class: 'form-horizontal' } do |f|
   .modal-body
     %div{class: modal? ? 'modal-body' : ''}
-      = f.input :identity, required: true, icon_hint: identity_hint
+      = f.input :identity, required: true, icon_hint: identity_hint, default: "fake1@gmail.com"
```

Comment:
Do you really want to display fake1@gmail.com as default value

Reply:
removed.